src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/EmbedBlockLinkViewModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using JsonKnownTypes;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
+
+[JsonKnownThisType(nameof(EmbedBlockLink))]
+public class EmbedBlockLinkViewModel : IContentBlockViewModel
```

Comment:
Might be worth exposing `created` / `updated` fields here if they end up getting added

Reply:
I discussed this with Ben and Nick. I'll add the `Created` and `Updated` fields to `EmbedBlock`.  There are already `Created` and `Updated` fields on `ContentBlock`. We discussed whether it was worth surfacing these into `IContentBlockViewModel`, but since the frontend doesn't use these values we decided we wouldn't. I think adding these columns is mostly to help us debugging and future proofing if they're needed in the future.

Reply:
Cool, initially just thought of this for the debugging purposes